Framework/PythonInterface/mantid/plots/mantidaxes.py

```diff
@@ -913,20 +913,23 @@ def _redraw_colorplot(self, colorfunc, artists_orig, workspace, **kwargs):
                     col.remove()
             if hasattr(artist_orig, 'colorbar_cid'):
                 artist_orig.callbacksSM.disconnect(artist_orig.colorbar_cid)
-        if artist_orig.norm.vmin == 0:  # avoid errors with log 0
```

Comment:
Is the norm.vmin check still needed?

Reply:
I wasn't really happy with this being at this low level and it looks like it stemmed from before we had checks at higher levels to stop bad input like that getting here. I tried to to test the behaviour but input of `0` is no longer allowed.